app/helpers/browse_tags_helper.rb

```diff
@@ -169,6 +171,17 @@ def telephone_links(_key, value)
     nil
   end
 
+  def tag2link(key, value)
+    return unless TAG2LINK.key? key
+
+    url = if value =~ %r{^https?://}
+            value
+          else
+            TAG2LINK[key].gsub("$1", value)
+          end
```

Comment:
In the not-so-unlikely-case that a mapper puts multiple values in there, wouldn't it be sensible to just display the value if it contains a semicolon?
Because a link to `facebook com / value1;value2` would almost certainly not yield an usable result either, would it?